brickpi3/brickpi3_spi.c

```diff
@@ -24,13 +24,13 @@
 
 #include "brickpi3.h"
 
-#define BRICKPI3_REQUIRED_FIRMWARE_VERSION	1002000 /* 1.2.0 */
+#define BRICKPI3_REQUIRED_FIRMWARE_VERSION	1004000 /* 1.2.0 */
 #define BRICKPI3_HEADER_SIZE		4
 #define BRICKPI3_ID_MSG_SIZE		16
 #define BRICKPI3_STRING_MSG_SIZE	20
 #define BRICKPI3_MAX_MSG_SIZE (BRICKPI3_HEADER_SIZE + BRICKPI3_STRING_MSG_SIZE)
 
-#define BRICKPI3_READ_FAILED(b)	((b)[3] != 0xA5)
+#define BRICKPI3_GET_FAILED(b)	((b)[3] != 0xA5)
```

Comment:
This macro is something I created for the ev3dev drivers and it is not mentioned in brickpi3.py (you can have look at the usage of 0xA5 in brickpi3.py to see what I mean)